omero/sysadmins/unix/server-installation.txt

```diff
@@ -212,12 +212,14 @@ The following Python packages are required:
       - `PyTables page <https://pytables.github.io/downloads.html>`_
 
 .. [1] The currently supported version of the django module used by
-       OMERO.web (1.8) requires Python 2.7.  The older version (1.6)
+       OMERO.web (1.8) requires Python 2.7. The older version (1.6)
        will work with Python 2.6 but lacks security support, and is
        consequently *not recommended for production use*. Python 2.7
        and `Django 1.8`_ are required for security support.
 
-.. [2] Make sure to have `libjpeg <http://libjpeg.sourceforge.net/>`_ installed when building `Pillow`_.
+.. [2] Make sure to have `libjpeg <http://libjpeg.sourceforge.net/>`_ 
+       installed when building `Pillow`_. Version 3.0.0 is currently
+       not supported.
```

Comment:
Figure is becoming a part of an install
@kennethgillen what do you think? 
Restrict now or leave it until figure is installed. but that could lead to error that people might not understand
We have also put a cap in all the files we used in the doc (cf. omero-install)